portal/.env.development

```diff
@@ -12,6 +12,18 @@ NEXT_PUBLIC_REQUEST_DEVICE_FORM_URL="https://enketo.samagra.io/::M34oPF2I"
 NEXT_PUBLIC_FAQ_DOCUMENT_URL="https://drive.google.com/file/d/1uKVDD2_c94I3qU85VEJrlKx4I8VDd0lp/view"
 NEXT_PUBLIC_GA_TAG_ID="G-9QL603X5R9"
 
+# FUSION AUTH
```

Comment:
Please remove these secrets from `.env.development` and add them to `.env.development.local`